src/services/FileUploadService.js

```diff
@@ -0,0 +1,21 @@
+import {
+  EventHandlerService,
+  PDFLEvents,
+} from "../services/EventHandlerService.js";
+/**
+ * Function to publishes the showReaderView event, reads a file and then publishes readNewFile event
+ * @param {pdfDoc} file PDF document
```

Comment:
Param is wrong, file is a param not pdfDoc

Reply:
fixed

Reply:
Yeah file type is File and documentation should be better since you mentioned both onReadNewFile and onShowReaderView event why not mention onResetReader event as well? Also you can drop "Function to" part and start your sentence with something like "Publishes ..."